src/drive/mobile/modules/settings/Settings.jsx

```diff
@@ -51,7 +51,7 @@ class Settings extends Component {
               isDebug={isDebug}
               sendFeedback={this.displayFeedbackForm}
             />
-            <About onTap={this.incrementTapCount} />
+            <About onTap={this.incrementTapCount} t={t} />
```

Comment:
This is my fault, but I think that doing a translate() within the About component is a better approach?